behave_django/environment.py

```diff
@@ -48,6 +49,17 @@ def load_fixtures(self, context):
         If fixtures are found in context, loads the fixtures using the loaddata
         management command.
         """
+        for step in context.scenario.all_steps:
+            match = step_registry.registry.find_match(step)
+            if match and hasattr(match.func, 'fixtures'):
+                if not hasattr(context, 'fixtures'):
+                    context.fixtures = []
+
+                # Load callables, and set fixture-strings to the context.
```

Comment:
This comment should not be necessary. Obviously, the code below is somewhat too cryptic. Can we make it more self-explanatory? e.g.
```python
fixtures = match.func.fixtures.setup(context=context) or []
context.fixtures += fixtures
```
Though, I'm not really sure what the `fixtures` is that the `setup()` method returns. (What would be a self-explanatory name instead?)

Reply:
Naming can and will be more concise once only fixtures (and not callables) are included in the decorator.

Reply:
That would make me sad, as I prefer callables over static fixtures (as you do).